forge-gui/res/editions/Alchemy Horizons Baldur's Gate.txt

```diff
@@ -93,7 +84,7 @@ ScryfallCode=HBG
 72 R Jon Irenicus, the Exile @Igor Grechanyi
 73 U Liara of the Flaming Fist @David Rapoza
 74 U Minthara of the Absolute @Evyn Fong
-75 M Tasha, Unholy Archmage @Martina Fackova
+75 M Tasha, Unholy Archmage @Martina Fačková
```

Comment:
Forge has had issues historically with these types of characters. I wouldn't add these without careful consideration

Reply:
Ok, I saw other Artist name with accent/diacritic so I thought is was ok. But if you tell me its not, I can remove it, no problem.

Reply:
There was a change at some point that supported this

Reply:
Indeed, "Martina Fačková" is present in several other edition file with those characters